app/controllers/api/v0/shipments_controller.rb

```diff
@@ -79,8 +79,9 @@ def add
       def remove
         variant = scoped_variant(params[:variant_id])
         quantity = params[:quantity].to_i
+        restock_item = !params[:restock_item].nil? ? params[:restock_item] == "true" : true
```

Comment:
I find this very difficult to read. Wouldn't you achieve the same with this?
```suggestion
        restock_item = params[:restock_item] == "true"
```

Reply:
I initially had the expression like that. But if for example the endpoint was used in the future for whatever reason but without `:restock_item` param, it would evaluate null == "true" which would set restock_item to false. The default behaviour is to ensure that restock_item is true, so the explicit check was added

Reply:
There you go, I read the negation wrong. I thought that `!` was negating the whole expression like this:
```rb
        restock_item = !(params[:restock_item].nil? ? params[:restock_item] == "true" : true)
```

If you want to test if the param is defined, it would be clearer to write this:

```rb
        restock_item = params.key?(:restock_item) ? params[:restock_item] == "true" : true
```

Negations are always confusing. Another nice Rails way is:

```rb
        restock_item = params[:restock_item].present? ? params[:restock_item] == "true" : true
```

That would see an empty string as not present. But your intention here is to provide a default in case the param isn't there. And there's a much better way for that:

```suggestion
        restock_item = params.fetch(:restock_item, "true") == "true"
```

Reply:
Thank you! The fetch is what I was looking for. It makes it soo much cleaner 👍